packages/cli/contracts/test/tablegen.t.sol

```diff
@@ -2,12 +2,11 @@
 pragma solidity >=0.8.0;
 
 import "forge-std/Test.sol";
-import {Table1, Table1Data} from "../test-contracts-src/tables/Table1.sol";
-import {Enum1, Enum2} from "../test-contracts-src/types.sol";
-
-contract tablegenTest is Test {
-  function isStore() external view {}
+import {StoreView} from "@latticexyz/store/src/StoreView.sol";
+import {Table1, Table1Data} from "../src/tables/Table1.sol";
+import {Enum1, Enum2} from "../src/types.sol";
 
+contract tablegenTest is Test, StoreView {
```

Comment:
Nit, I'd always capitalize contract names (we should also capitalize the file then)
```suggestion
contract TablegenTest is Test, StoreView {
```

Reply:
updated